### DISCUSSION TITLE
Release v1.0.8

### DISCUSSION THREAD

**Discussion by BazzaCuda** [BazzaCuda/CustomMenu#21] [Announcements]
Windows 64-bit binaries

- fixed an issue where **CustomMenu** wasn't recording the Ctrl/Alt/Shift modifiers when the user selects a hotkey. As a result, the modifier combination wasn't being applied the next time **CustomMenu** was run. If you want to add a Ctrl/Alt/Shift modifier combination to your hotkey, you will need to delete the contents of the "hotkey" file and relaunch **CustomMenu**.

<hr /><em>This discussion was created from the release <a href='https://github.com/BazzaCuda/CustomMenu/releases/tag/v1.0.8'>Release v1.0.8</a>.</em>

**Comment by kksshh231** [comment]
Can't I set a relative path?

**Reply by BazzaCuda** [reply]
Yes, you can.

The easiest way is to drag and drop something from Windows onto the menu tree of the Config window while holding down a SHIFT key - the hint will change from "Create with absolute paths" to "Create with relative paths".

**CustomMenu** will automatically set the Command, Directory and Icon File boxes based on the relative path to the target from the location of CustomMenu.exe.

I would suggest dragging and dropping anything as a test to see how it's done.

This allows you to run **CustomMenu** from a USB stick where you can't be 100% sure which drive letter Windows will assign to it during Plug & Play.

Any questions, let me know.

Do you like the hotkey functionality I added per your request? ;)

**Reply by kksshh231** [reply]
Thanks to BazzaCuda, I am using CustomMenu well. Thank you so much.
I also like the shortcut function. CustomMenu has become increasingly easier to use.
great. Also, when you run CustomMenu and select the program from the right menu to run it, the mouse point stops and then moves again.

**Comment by BARBOUDIS** [comment]
CONGRATULATIONS for this amazing work of yours and thank you so much! 

There's only one single matter that I couldn't resolve by my own... 

![Χωρίς τίτλο](https://github.com/BazzaCuda/CustomMenu/assets/150555483/fad8dcbb-c1ad-43f3-8f32-1e0696c5763d)

In my case, when I click "Write to Desktop Menu" (after clicking ''Save'' of cource, otherwise is unavailable) it asks me if I agree about the backup .reg file... I click ''Yes'', it's saving (loading) it and when it finishes I'm right clicking on my desktop, my Custom Menu appears BUT the Windows Desktop Context Menu is missing... 

Like this: 

![Χωρίς](https://github.com/BazzaCuda/CustomMenu/assets/150555483/d3474966-e665-4520-9db4-6758fd962fd3)

Any ideas why?

**Reply by BazzaCuda** [reply]
Hi.

Firstly, thank you for your kind words about Custom Menu. 

Secondly, let me explain:

If you right-click on the desktop, you will get the menu that you built using Custom Menu.

However, if you hold down a **_CTRL_** key and right-click the desktop, you will get the standard Windows desktop context menu. As you have done a "Write to desktop menu" as you described, the top part of the menu will be the same as your Custom Menu and, lower down, Windows will have added any shell extensions that are registered on your machine, such as the standard "View", "Sort by", "Refresh" options, or the "New" menu item with all it's sub-menu options, like "New Text Document"; there will likely be others depending on what other software you have installed on your machine.

Also, if you hover your mouse over an icon on your desktop, hold down a CTRL key and right-click the icon, you will get the standard desktop menu for that icon, with items such as this:
![image](https://github.com/BazzaCuda/CustomMenu/assets/22550919/faade741-05c1-4f0d-b720-38b3d99c1f90)

_Please note, if you have used the "Browse" function in CustomMenu to create a live menu of a folder on your machine, this can only be used from your Custom Menu. Clicking the same item in the Windows menu doesn't work as Windows doesn't understand the browse command._

You can also right-click just above the taskbar (within 30 pixels); this will also display the Windows desktop menu rather than your Custom Menu.

Alternatively, rather than right-clicking the desktop, you can create a hotkey to activate Custom Menu. This is useful when you have lots of windows open and can't see your desktop. If you read back through the releases, there are some easy instructions on how to create a hotkey. If it's not clear, let me know and I'll explain.

Let me know how you get on.

**Reply by BARBOUDIS** [reply]
Thanks, mate... 

Right-click just above the taskbar is SUPER DOOPER!

I've already created a hotkey...

My proposal / request (for a possible future upgrade) is about holding down a CTRL key and right-click an icon (to get the standard desktop menu for it) 

Maybe we can have it like ONLY JUST RIGHT-CLICKING AN ICON without the CTRL key... 

I personally believe that it would be THE PERFECT MATCH... 

Anyway, thank you again for your interest and God bless!

**Comment by Arjay-El** [comment]
Win11 23H2. MS's next version to continue the punishment. Right-click does not open app. Neither does double clicking or selecting open from the explorer menu. Any suggestions?

**Reply by Arjay-El** [reply]
You need a box with a "Dummies Guide" heading. If I right click just above the taskbar, it opens up. I was right clicking the icon. Ignore my question. Thanks.

**Reply by BazzaCuda** [reply]
Glad you're all sorted now.
